src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1602,6 +1676,19 @@ subroutine wrfv3_netcdf(fv3filenamegin)
     real(r_kind),allocatable,dimension(:,:,:)::ges_delzinc
     integer(i_kind) k
 
+    real(r_kind),pointer,dimension(:,:,:):: ges_ql  =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_qi  =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_qr  =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_qs  =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_qg  =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_qnr =>NULL()
+    real(r_kind),pointer,dimension(:,:,:):: ges_w   =>NULL()
+
+    real(r_kind), dimension(lat2,lon2,nsig) :: io_arr_qr, io_arr_qs
+    real(r_kind), dimension(lat2,lon2,nsig) :: io_arr_qg, io_arr_qnr
+
+    integer(i_kind) i,j
```

Comment:
The variables, i and j, have been identified as unused.  Please remove unused variables.

Reply:
We deleted them as you suggested.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-4c910ebd862ea8bea4024bbfa73835e643efb4091155d4e8ef121388b14ab6ffR1677